papis/bibtex.py

```diff
@@ -13,35 +13,109 @@
 
 logger = logging.getLogger("bibtex")  # type: logging.Logger
 
-bibtex_types = [
-    "article", "book", "booklet", "conference", "inbook", "incollection",
-    "inproceedings", "manual", "mastersthesis", "misc", "phdthesis",
-    "proceedings", "techreport", "unpublished"
-] + papis.config.getlist("extra-bibtex-types")  # type: List[str]
+# NOTE: see the BibLaTeX docs for an up to date list of types and keys:
+#   https://ctan.org/pkg/biblatex?lang=en
+
+bibtex_types = frozenset([
+    # regular types (Section 2.1.1)
+    "article",
+    "book", "mvbook", "inbook", "bookinbook", "suppbook", "booklet",
+    "collection", "mvcollection", "incollection", "suppcollection",
+    "dataset",
+    "manual",
+    "misc",
+    "online",
+    "patent",
+    "periodical", "suppperiodical",
+    "proceedings", "mvproceedings", "inproceedings",
+    "reference", "mvreference", "inreference",
+    "report",
+    # "set",
+    "software",
+    "thesis",
+    "unpublished",
+    # "xdata",
+    # "custom[a-f]",
+    # non-standard types (Section 2.1.3)
+    "artwork",
+    "audio",
+    "bibnote",
+    "commentary",
+    "image",
+    "juristiction",
+    "legislation",
+    "legal",
+    "letter",
+    "movie",
+    "music",
+    "performance",
+    "review",
+    "standard",
+    "video",
+]) | frozenset(papis.config.getlist("extra-bibtex-types"))  # type: FrozenSet[str]
 
 bibtex_type_converter = {
```

Comment:
I used these `converter` dicts for the aliases defined in the manual, but I'm not quite sure what the entries that were already there were.

@alejandrogallo  Do you happen to recall where does come from?

Reply:
no, sadly I don't remember quite, this was some years ago, sorry

Reply:
Did a bit of `git` blaming and it turns out this was introduced in 
https://github.com/papis/papis/pull/121
to handle some Zotero fields. Some of these do still appear in the Zotero schema
https://api.zotero.org/schema

I'll add a note for the future :D

Reply:
that's wise ;)